keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -2604,22 +2743,39 @@ skew are less likely to occur.
     - kubelet
     - kube-scheduler
     - kube-controller-manager
+- [X] Feature gate
+  - Feature gate name: DRAAdminAccess
+  - Components depending on the feature gate:
+    - kube-apiserver
+
+
 
 ###### Does enabling the feature change any default behavior?
 
 No.
 
+DRAAdminAccess: no.
+
 ###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
 
 Yes. Applications that were already deployed and are running will continue to
 work, but they will stop working when containers get restarted because those
 restarted containers won't have the additional resources.
 
+DRAAdminAccess: No. Workloads which were deployed with admin access will continue
```

Comment:
But if it's in a `ResourceClaimTemplate`, then Pod recreation will fail, right?

Reply:
If the claims still need to be generated, then we have a different failure scenario: the template has management access, the claim won't.

This calls for yet another feature gate check, this time in the claim controller: if a template has management access set and the feature gate is off, refuse to generate the claim. Added.

So the answer here becomes "it depends". Updated.